airbyte-webapp/src/components/connectorBuilder/Builder/BuilderSidebar.tsx

```diff
@@ -73,6 +73,7 @@ export const BuilderSidebar: React.FC<BuilderSidebarProps> = React.memo(({ class
       onSubmit: () => {
         setValues(DEFAULT_BUILDER_FORM_VALUES);
```

Comment:
Doesn't need to be fixed on this PR, but it seems like this doesn't work reliably - after hidden "Reset all", then reloading the page I'm getting back to the previous state (there is probably another render-cycle of some kind putting it back). A separate page with a separate context for state management (see comment above) might help with this automatically.

Reply:
See earlier comment, after the refactor and making this properly update local storage, this should now be fixed.